### DISCUSSION TITLE
Styling the -mseg content

### DISCUSSION THREAD

**Discussion by the-argus** [davatorium/rofi#1762] [Theming]
How can I select and style the -mesg content in a theme? I tried styling the ``msg`` and ``mesg`` elements, but neither appeared to be correct.

**Comment by DaveDavenport** [comment]
https://github.com/davatorium/rofi/blob/next/doc/rofi-theme.5.markdown#basic-structure-1
That should explain the names of the widgets.

**Comment by the-argus** [comment]
Thanks for the quick response, I've got my theme working now.  I'd say that -mesg should be renamed to -message in that case, or at least add a little not in the documentation that the widget created by the -mesg flag is called message.

**Reply by DaveDavenport** [reply]
I don't think renaming the option is wise, it would break any script using the option.

The documentation for -mesg says it adds a message line, I would assume that is clear enough to make the association:

```
       -mesg string

       Add a message line below the filter entry box. Supports Pango markup.  For more information on supported markup, see here ⟨https://docs.gtk.org/Pango/pango_markup.html⟩
```
